app/src/main/res/values/strings.xml

```diff
@@ -131,7 +131,12 @@
     <string name="channel_state_pending_force_closing">pending force close</string>
     <string name="channel_state_waiting_close">waiting close</string>
     <string name="no_channels">No channels</string>
-
+    <string name="channel_closing_transaction">Closing Transaction</string>
+    <string name="channel_close">Close Channel</string>
+    <string name="channel_close_force">Force Close Channel</string>
+    <string name="channel_close_error">Error occurred while closing channel</string>
+    <string name="channel_close_confirmation">Do you really want to close the channel with:\n%s</string>
```

Comment:
I would actually add a \n at the end as well. Makes the message more readable in my opinion